styles.css

```diff
@@ -3,57 +3,72 @@
 body {
   background-color: gainsboro;
   font-family: "Poppins", sans-serif;
+  max-width: 100vw;
 }
 
 .container {
   display: flex;
+  background-color: gainsboro;
   flex-direction: column;
   align-items: center;
 }
 
-.upperNavbar {
+.logo {
+  margin-right: 180px;
+  text-decoration: none;
+  height: 76px;
+  width: 20px;
+  margin-top: 10px;
+  margin-left: 12px;
+  color: #6070ff;
+}
+
+.header {
+  height: 15px;
+  width: 375px;
+}
+
+.navbar {
   background-color: white;
-  margin: 20px 20px 0 20px;
-  padding: 29px 29px 0 29px;
+  margin-top: 10px;
```

Comment:
- Kindly remove this margin-top to your nav bar it also adds the extra space on top of your navigation which is not needed. kindly fix that it will make your design match more to the Figma design as required in the project :sunglasses: :sunglasses: :sunglasses:

Reply:
Hello @MaganaAsiati 

-Fix margin-top issue
-Fix the background-color issue

-Can you a little more explanation about the padding and margin in lines 16 and 17 respectively, as I don't have them in those lines?

Thank you for the review

Reply:
Hello @MaganaAsiati 

-Fix the Design issue

Thank you for the review

I would you to recheck it now, please